reactor-test/src/main/java/reactor/test/util/RaceTestUtils.java

```diff
@@ -99,77 +101,79 @@
 		}
 	}
 
+	public static void race(final Runnable r1, final Runnable r2) {
+		race(new Runnable[]{r1, r2});
+	}
+
 	/**
-	 * Synchronizes the execution of two {@link Runnable} as much as possible
-	 * to test race conditions. The method blocks until both have run to completion.
-	 * @param r1 the first runnable
-	 * @param r2 the second runnable
+	 * Synchronizes the execution of several {@link Runnable}s as much as possible
+	 * to test race conditions. The method blocks until all have run to completion.
+	 * @param rs the runnables to execute
 	 */
-	public static void race(final Runnable r1, final Runnable r2) {
-		race(r1, r2, Schedulers.single());
+	public static void race(final Runnable... rs) {
+		race(Schedulers.parallel(), rs);
```

Comment:
I don't know if using the global `parallel()` is such a good idea here. `Schedulers.single()` was already debatable I think, but `Schedulers.parallel()` has a higher chance of side effects due to the fact that this is the default implicit `Scheduler` for all core operators that require a `Scheduler`.

Reply:
`Schedulers.single()` was okay-ish since the other runnable was running on the calling thread. I tried with bounded elastic at first, but encountered some problems which may actually have been unrelated, so may give it another try. Or maybe `Schedulers.elastic()`, wdyt?

Reply:
let's aim at reintroducing the `Schedulers.boundedElastic()` here 🙏